pkg/cmd/kafka/kafkaerror/error.go

```diff
@@ -0,0 +1,34 @@
+package kafkaerror
```

Comment:
This does not belong in pkg/cmd (should be reserved for commands). Check in `pkg/api`

Reply:
I've moved it there.